pkgs/applications/audio/reaper/default.nix

```diff
@@ -1,10 +1,35 @@
-{ lib, stdenv, fetchurl, autoPatchelfHook, makeWrapper
-, alsaLib, xorg, libjack2
-, gtk3, pango, gdk-pixbuf, cairo, glib, freetype
-, libpulseaudio, xdg_utils
+{ config, lib, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+
+, alsaLib
+, gtk3
+
+, jackSupport ? true, libjack2
+, pulseaudioSupport ? config.pulseaudio or true, libpulseaudio
+, mp3Support ? true, lame
+, ffmpegSupport ? true, ffmpeg
+, vlcSupport ? true, vlc
```

Comment:
Do we really need options to disable mp3, ffmpeg and vlc support? Such options are normally not tested regularly and easily bit rot over time.